db/migrate/20220917044313_remove_author_id_from_user.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAuthorIdFromUser < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :AuthorId, :integer
+  end
+end
```

Comment:
- [ ] Kindly note that you are having errors in your migration, you are trying to access attributes that don't exist in some table, kindly fix them

![image](https://user-images.githubusercontent.com/65068771/190843945-119fc889-cbbf-49ef-aeaa-3d65ae9e60e3.png)

Reply:
@andersonlebon 
Yes, you're right bro. I have already deleted the AuthorId column. how it can be present. I have already migrated that.
Waiting for your reply